lib/decanter/version.rb

```diff
@@ -1,3 +1,3 @@
 module Decanter
-  VERSION = '3.6.0'.freeze
+  VERSION = '3.6.1'.freeze
```

Comment:
I think this might qualify as a breaking change. While it is technically a bug, at this point there could be apps relying on it as a feature.